README.md

```diff
@@ -22,6 +22,30 @@ $ npm install eslint-plugin-googleappsscript --save-dev
 
 ## Usage
 
+### ESLint flat config
+
+Add `import googleAppsScript from 'eslint-plugin-googleappsscript';` to `eslint.config.mjs`,
```

Comment:
```suggestion
Add `import googleAppsScript from 'eslint-plugin-googleappsscript';` to `[eslint.config.(js|mjs|cfs)](https://eslint.org/docs/latest/use/configure/configuration-files)`,
```
Perhaps it would be better to list the different file types here and link to the docs.